chainer/link.py

```diff
@@ -194,6 +194,15 @@ def __init__(self):
         finally:
             self._within_init_scope = old_flag
 
+    def __call__(self, *args, **kwargs):
+        try:
+            forward = self.forward
+        except AttributeError:
+            raise TypeError(
+                    '{} object does not have neither \'__call__\' nor '
+                    '\'forward\' method.'.format(self))
```

Comment:
This sentence is not accurate (`self` has `__call__`).

"Neither Link.\_\_call__() method is overridden nor forward() method is defined."?

Reply:
Thanbks, fixed